plugins/txt/plugin/commands/txt_new_scene.rb

```diff
@@ -0,0 +1,66 @@
+module AresMUSH
+  module Txt
+      class TxtNewSceneCmd
+        include CommandHandler
+
+        attr_accessor :names, :names_raw, :message, :scene_id
+
+    def parse_args
+      if (!cmd.args)
+        self.names = []
+      else
+        args = cmd.parse_args(ArgParser.arg1_equals_arg2)
+        if (args.arg1 && (args.arg1.include?("http://") || args.arg1.include?("https://")) )
```

Comment:
Pretty sure that was taken from page code so that you don't accidentally... something with a link? I'm pretty sure it's not needed here, because you can't txt/newscene someone without both args. I removed it and added required_args instead.

Reply:
Oh, that's in the page code because sometimes you try to page somebody a link, and the link has an "=" in it, and if I just do `p Here's some link http://somewhere=foo` it'll think that you're trying to page a character named "Here's some link http://somewhere"  :)   I don't know if you need that in the text code or not.